src/adaptations/device-layer/include/Weave/DeviceLayer/ConfigurationManager.h

```diff
@@ -111,6 +123,11 @@ class ConfigurationManager
     bool IsPairedToAccount();
     bool IsMemberOfFabric();
     bool IsFullyProvisioned();
+    bool DeviceCredentialsProvisioned();
+    bool DeviceICACertsProvisioned();
+    bool ManufAttestDeviceICACertsProvisioned();
+    bool UseManufAttestCredentialsAsOperational();
+    void UseManufAttestCredentialsAsOperational(bool val);
```

Comment:
When do we envision this being used?  In particular, if a device has op credentials, when would we _not_ use them?

If we do keep this, I'm not sure we want this to be public.